content/api/vision/marker-ids.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Marker IDs
+---
+
+Each marker used in the game has an ID that corresponds to the marker type. You can use this within your code to determine what sort of marker you're looking at and make decisions based off of it.
+
+There are some useful constants which can be imported to do this. For example:
+
+```python
+from robot import WALL, TOKEN_ZONE_1, COLUMN_N_FACING_N
+```
```

Comment:
Since there seems to be a lot of constants like these, I think here would be a good place to mention `from robot import *`.

Reply:
I really dislike this pattern, as no only is it not obvious if / where something it defined, it also makes linting impossible

Reply:
Fair enough, that's fine.